packages/api/src/feature_flag/controller.test.ts

```diff
@@ -283,3 +287,182 @@ describe("POST /feature-flag", () => {
     expect(logger.error).toHaveBeenCalled();
   });
 });
+
+describe("PUT /feature-flag/1bdf2da6-026b-4e8e-9b57-a86b1817be5d", () => {
```

Comment:
Nit: the URL param should be given as `:featureFlagId` here, rather than using a specific value